lib/WUI/wui_api.cpp

```diff
@@ -17,6 +17,7 @@
 #include "stm32f4xx_hal.h"
 #include "print_utils.hpp"
 #include "marlin_client.hpp"
+#include "marlin_client.h"
```

Comment:
Wait a moment. Do we have two marlin_client headers? That seems super-weird.

Reply:
Apparently we do :man_shrugging: , but the `.h`  is included in `.hpp`, so the include here is redundant.